### PR TITLE
Upgradle gradle version to 6.8.3

### PR DESCRIPTION
**Pull request by usmansaleem**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 #Changelog
 
+## 1.0.16
+### Features Added
+- No code changes from 1.0.15
```
Comment (jframe) [inline CHANGELOG.md:5]: Seems unnecessary to mention there are no code changes

File: gradlew [hunk 2]
```diff
@@ -82,6 +82,7 @@ esac
 
 CLASSPATH=$APP_HOME/gradle/wrapper/gradle-wrapper.jar
 
+
```
Comment (jframe) [inline gradlew:85]: nit: Why all the additional whitespace?

**Review by jframe — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (usmansaleem) [reply CHANGELOG.md:5]: 👍 I'll remove these lines.

File: gradlew (hunk 2 above)
Comment (usmansaleem) [reply gradlew:85]: that's auto-generated by gradle wrapper command.

**Pull request merged**